### PR TITLE
Remove JSON support

### PR DESCRIPTION
**Pull request by cb372**
Get rid of the `useThrift` constructor param and make the client always use Thrift.

Also get rid of the `Limits` object.

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -81,9 +81,6 @@ val CapiModelsVersion = "9.21"
 
 libraryDependencies ++= Seq(
   "com.gu" % "content-api-models-scala" % CapiModelsVersion,
-  "com.gu" % "content-api-models-json" % CapiModelsVersion,
```
Comment (LATaylor-guardian) [inline build.sbt]: This is only used by Concierge now if I'm correct?

**Comment by LATaylor-guardian** [comment]
Looks good to me 👍

File: build.sbt (hunk 1 above)
Comment (cb372) [inline build.sbt]: Yes, I think so. It could be moved inside Concierge in the future.

**Comment by nicl** [comment]
@cb372 @LATaylor-guardian are you removing JSON support from the API, or just the client?

**Comment by LATaylor-guardian** [comment]
@nicl Just the client so it uses Thrift by default. This should be transparent to any users of the Scala client.

**Comment by nicl** [comment]
@LATaylor-guardian okay thanks!

**Pull request merged**